Core.Tests/StateManagerTests.cs

```diff
@@ -2,12 +2,9 @@
 
 namespace Core.Tests
 {
-    public class StateManagerTests
-    {
-        [Fact]
-        public void ReturnsTrue()
-        {
-            Assert.True(true);
-        }
-    }
+	public class StateManagerTests
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931949)